core/boot.go

```diff
@@ -211,7 +219,12 @@ func ConnectDB() (*sqlx.DB, error) {
 // LoadConf plugins.json into a usable struct.
 func LoadConf() (*PluginJSON, error) {
 	ipath := "github.com/itsabot/abot"
-	p := filepath.Join(os.Getenv("GOPATH"), "src", ipath, "plugins.json")
+	path := os.Getenv("GOPATH")
+	if len(path) == 0 {
+		log.Fatal(PathError)
```

Comment:
What do you want to do then? Just log the error? If GOPATH can't be found, abot will break...